lookout/style/typos/tests/test_preprocessing.py

```diff
@@ -26,6 +27,21 @@ def test_filter_splits(self):
                                   index=[0, 1, 5])
         assert_frame_equal(filter_splits(data, vocabulary), result)
 
+    def test_print_frequencies(self):
+        frequency_column = "num_occ"
+        id_stats = pandas.DataFrame([[23], [6], [17]], columns=[frequency_column],
+                                    index=["get", "token", "set"])
+        vocabulary = {"get", "set"}
+        with tempfile.NamedTemporaryFile() as vocabulary_file:
+            print_frequencies(vocabulary, id_stats, frequency_column, vocabulary_file.name)
+            with open(vocabulary_file.name, "r") as f:
```

Comment:
```suggestion
            with open(vocabulary_file.name) as f:
```

I found this in several more places in the code, please change all the rest.